Assignment2/Mail.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+// Load Composer's autoloader.
+require './vendor/autoload.php';
+
+// Import PHPMailer classes into the global namespace.
+// These must be at the top of your script, not inside a function.
+use PHPMailer\PHPMailer\PHPMailer;
+use PHPMailer\PHPMailer\SMTP;
+use PHPMailer\PHPMailer\Exception;
+
+if (isset($_POST['submit'])) {
+  $email = $_POST['email'];
+
+  // Create an instance.
+  $mail = new PHPMailer(true);
```

Comment:
Please implement Object-Oriented Programming (OOP) concepts here. There may be multiple scenarios where sending an email is necessary, so let's create a class with a corresponding function for reusability whenever needed.